### PR TITLE
Remove 1010cg feature toggle

### PR DESCRIPTION
**Pull request by LinnJS**
## Description
Since we are now finished staggering the release of the 10-10CG with great success we should remove the unneeded feature flip to increase ease of workflow and decrease code complexity.

[Remove FeatureFlip from 10-10CG Ticket](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/18468)

## Testing done
- Manual testing checked the RI to make sure it worked as expected http://6485b4171c278886ffab88c425cf703a.review.vetsgov-internal/family-member-benefits/comprehensive-assistance-for-family-caregivers/

## Screenshots
N/A

## Acceptance criteria
- [ ] 10-10CG Feature flip used to turn form on/off is removed

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by Mottie — APPROVED** [review]
ghe-emoji

**Review by U-DON — APPROVED** [review]

File: src/applications/static-pages/caregiver-content-toggle/widget.jsx
Comment (ncksllvn) [inline src/applications/static-pages/caregiver-content-toggle/widget.jsx:7]: Since this is ultimately going to be completely static content, I think we should let the editors know so that we can just add that content onto the actual page in the CMS, then we can just delete this whole directory

**Review by ncksllvn — APPROVED** [review]
This LGTM but we may want to hold off til EOD to see if the content team can just remove the React widget from the page altogether. Then we can just remove all of this code. 

Thread in case you want to follow, https://dsva.slack.com/archives/C0G5ZFK5J/p1611760848001900

File: src/applications/static-pages/caregiver-content-toggle/widget.jsx
Comment (LinnJS) [reply src/applications/static-pages/caregiver-content-toggle/widget.jsx:7]: @ncksllvn this was on hold for weeks and never got the go-ahead from stakeholders... so I was planning on asking once we got approval, thanks for reaching out!

**Review by ncksllvn — APPROVED** [review]
LGTM! 
image

**Comment by ala-yna** [comment]
@LinnJS just reviewed the RI & files changed - looks good to me!

**Pull request merged**
